internal/vds/vds_test.go

```diff
@@ -767,9 +614,9 @@ func TestSurfaceUnalignedWithSeismic(t *testing.T) {
 		interpolationMethod,
 	)
 	require.Len(t, buf, len(targetAttributes), "Wrong number of attributes")
-	require.NoError(t, err, "Failed to fetch horizon")
+	require.NoErrorf(t, err, "Failed to fetch horizon")
 	result, err := toFloat32(buf[0])
-	require.NoError(t, err, "Failed to covert to float32 buffer")
+	require.NoErrorf(t, err, "Failed to covert to float32 buffer")
 	require.Equalf(t, expected, *result, "Horizon not as expected")
 }
 
```

Comment:
Update "TestSurfaceWindowVerticalBounds" too?

Reply:
I don't want to use `require.Equal(t, err, testcase.Err)` here if that's what you mean. Then I have to hardcode the exact error message into each test case